libs/networking/Testing_client.mli

```diff
@@ -0,0 +1,14 @@
+type test_response = { response : Cohttp_lwt.Response.t; body_path : string }
```

Comment:
maybe you can add some explanations in the .mli on how to use this library. Maybe the code of a full test using
those mocks.